site-src/references/implementations.md

```diff
@@ -39,6 +41,14 @@ Note that not all of the `v0.2.0` specification is completed yet, [contributions
 [contour-1]:https://projectcontour.io/guides/gateway-api/
 [contour-2]:https://github.com/projectcontour/contour/blob/main/CONTRIBUTING.md
 
+### Emissary-Ingress (Ambassador API Gateway)
```

Comment:
Can you add a link to documentation on how to use the API with Emissary?

Reply:
Done